.travis.yml

```diff
@@ -10,9 +10,9 @@ python:
 os:
     - linux
 stages:
+  - lint
```

Comment:
and this.

Reply:
Why do the linting at the end after all tests?
Usually, linting is done before tests

Reply:
nope

Reply:
we prefer unit first to give a faster review.

Reply:
ok